tests/tests/integration_tests.rs

```diff
@@ -296,7 +296,7 @@ async fn ws_subscription_several_clients_with_drop() {
 	let mut clients = Vec::with_capacity(10);
 	for _ in 0..10 {
 		let client = WsClientBuilder::default()
-			.max_buffer_capacity_per_subscription(u32::MAX as usize)
+			.max_buffer_capacity_per_subscription(u16::MAX as usize)
```

Comment:
This failed locally because it tried to allocate lots of memory.

https://docs.rs/tokio/latest/src/tokio/sync/broadcast.rs.html#501-534 it reserves the capacity which is different from the mpsc channel